api/src/logger.ts

```diff
@@ -44,6 +44,30 @@ if (env['LOG_STYLE'] !== 'raw') {
 		},
 	};
 }
+if (env['LOG_STYLE'] === 'raw') {
+	const ssoCookies = toArray(env['AUTH_PROVIDERS'] ?? '').flatMap((provider: string) => {
+		if (provider.length === 0) return [];
+		return [`oauth2.${provider.toLowerCase()}`, `openid.${provider.toLowerCase()}`];
+	});
+	httpLoggerOptions.redact = {
+		paths: ['req.headers.authorization', 'req.headers.cookie', 'res.headers'],
+		censor: (value, pathParts) => {
+			const path = pathParts.join('.');
+			if (path === 'req.headers.cookie') {
+				return redactHeaderCookie(value, [`access_token`, `${env['REFRESH_TOKEN_COOKIE_NAME']}`, ...ssoCookies]);
+			}
+			if (path === 'res.headers') {
+				if ('set-cookie' in value) {
+					value['set-cookie'] = toArray(value['set-cookie']).map((cookie) =>
+						redactHeaderCookie(cookie, [`access_token`, `${env['REFRESH_TOKEN_COOKIE_NAME']}`, ...ssoCookies])
+					);
+				}
+				return value;
+			}
```

Comment:
Let's extract `cookieNames`.

```suggestion
			const path = pathParts.join('.');
			const cookieNames = [`access_token`, `${env['REFRESH_TOKEN_COOKIE_NAME']}`, ...ssoCookies];
			if (path === 'req.headers.cookie') {
				return redactHeaderCookie(value, cookieNames);
			}
			if (path === 'res.headers') {
				if ('set-cookie' in value) {
					value['set-cookie'] = toArray(value['set-cookie']).map((cookie) => redactHeaderCookie(cookie, cookieNames));
				}
				return value;
			}
```

Reply:
Good one but lets pull the cookieNames list to the root so it only gets constructed once.